### ISSUE TITLE
Fine tune tricks

### ISSUE THREAD

**Issue by jexterliangsufe** [Time-MoE/Time-MoE#40]
Hi, greate works! I try to use TimeMoE-50M in my situation. At first I predict directly without fine-tuning. The result is worse than my full-shot TemporalFusionTransformer. Then I try to fine-tune the model with a (10w+, 365) dataset. The result is worse than the zero-shot result when using the default `max_length=1024`. Then I change the max_length to 128 which is equal to the `context_length`  used in the predicting procedure. The result seems to be close to the zero-shot result, but still worse than the TFT result. 
In your paper, I noticed that you set the number of fine-tune epochs to only one and it is the default setting in your codes. Are there any other fine-tune tricks?

**Comment by Maple728** [comment]
Thank you for your interest in our work!

Could you please provide the hyperparameters used for fine-tuning the model and the settings used for inference?

**Comment by jexterliangsufe** [comment]
I only revise `max_length` and `normalization_method` when fine-tuning. For inference, I use several `context_length` such as 30/64/96/128. Except for the above, everything is the default.

**Comment by Maple728** [comment]
Could you please provide more details about the training process, such as the training command and settings (e.g., batch size, total training steps) for fine-tuning TimeMoE? Additionally, could you share the inference settings, like prediction length and the corresponding context length? This information would help me gain a deeper understanding of the problem.

Note: The max_position_embeddings for Time-MoE is set to during training. This means the maximum sequence length for Time-MoE is 4096. To achieve optimal forecasting performance, it is recommended that the sum of context_length and prediction_length does not exceed 4096. If you wish to support longer sequence length, please fine-tune Time-MoE with the desired longer sequence length.

**Comment by schlinge2000** [comment]
Dear Maple728,
I am also pretty interested in fine-tuning the model couod you provide a code snippet to show, how to properly fine-tune the model?

**Comment by Maple728** [comment]
> Dear Maple728, I am also pretty interested in fine-tuning the model couod you provide a code snippet to show, how to properly fine-tune the model?

You can find the code snippet for fine-tuning Time-MoE [here](https://github.com/Time-MoE/Time-MoE?tab=readme-ov-file#-fine-tuning-time-moe). Use the appropriate code for your hardware setup, whether it's a CPU, single-node GPU, or multi-node GPUs.
